conda-recipes/tensorflow-datasets-feedstock/recipe/build.sh

```diff
@@ -21,7 +21,7 @@ git clone https://github.com/tensorflow/datasets.git
 cd datasets
 
 echo ${PKG_VERSION}
-TAGNAME=$(git tag --list | grep "${PKG_VERSION}")
+TAGNAME=$(git tag --list | grep "v${PKG_VERSION}")
```

Comment:
This change is needed because crazy tensorflow datasets repo has two tags for 1.1.0 (1.1.0 and v1.1.0). Latter one is the latest one and is needed by us.